### PR TITLE
Shortcut for "Following tags"

### PR DESCRIPTION
**Pull request by ludwiczakpawel**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

This PR adds a shortcut to access "Following tags" from the "Tags" page.

## QA Instructions, Screenshots, Recordings

image

## Added/updated tests?

- [ ] Yes
- [x] No, and this is why: not needed
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://developers.forem.com) and/or
      [Admin Guide](https://admin.forem.com/), or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] I've updated the README or added inline documentation
- [ ] I've added an entry to
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [ ] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [x] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

### CODE REVIEW

File: app/views/tags/index.html.erb [hunk 1]
```diff
@@ -20,8 +20,9 @@
 <% end %>
 
 <main id="main-content" class="crayons-layout crayons-layout--1-col">
-  <header class="px-2 m:px-0">
+  <header class="px-2 m:px-0 flex items-center justify-between">
     <h1>Top tags</h1>
+    <a class="crayons-btn crayons-btn--ghost" href="<%= dashboard_following_tags_path %>">Following tags</a>
```
Comment (Ridhwana) [inline app/views/tags/index.html.erb:25]: We should probably use the rails link_to method, something like this: 

```
<%= link_to "Following tags", dashboard_following_tags_path, class: "crayons-btn crayons-btn--ghost" %>
```

Comment (ludwiczakpawel) [reply app/views/tags/index.html.erb]: 👍 i've pushed the change!

File: app/views/tags/index.html.erb [hunk 2]
```diff
@@ -20,8 +20,9 @@
 <% end %>
 
 <main id="main-content" class="crayons-layout crayons-layout--1-col">
-  <header class="px-2 m:px-0">
+  <header class="px-2 m:px-0 flex items-center justify-between">
     <h1>Top tags</h1>
+    <%= link_to "Following tags", dashboard_following_tags_path, class: "crayons-btn crayons-btn--ghost" %>
```
Comment (nickytonline) [inline app/views/tags/index.html.erb:25]: nitpick (suggestion): The wording is not clear. Consider _Followed Tags_ or _Tags I Follow_.

**Review by Ridhwana — APPROVED** [review]
LGTM but I like @nickytonline's suggestion about amending the label

File: app/views/tags/index.html.erb (hunk 2 above)
Comment (ludwiczakpawel) [reply app/views/tags/index.html.erb:25]: this is the label we use in Dashboard so if we change it in one place we should do it in others too.. as i'm not an expert when it comes to proper English I'd rely on your suggestion but would love to double check if you think it also makes sense to update it in Dashboard.

Comment (nickytonline) [reply app/views/tags/index.html.erb:25]: I think it makes sense to change it there too. Followed Tags probably makes the most sense and it's fewer characters. Also good catch on keeping naming consistent!

**Review by nickytonline — APPROVED** [review]
Approving assuming the button and dashboard naming change. 🚀

File: app/views/tags/index.html.erb (hunk 2 above)
Comment (ludwiczakpawel) [reply app/views/tags/index.html.erb:25]: I'm looking at Twitter, Instagram and GitHub now and they all use "Following" in this scenario.. And I'm now officially confused :D

**Comment by ludwiczakpawel** [comment]
@nickytonline there's been an ongoing convo about naming "Following" vs "Followed" and since it's a bit controversial topic, I'm keeping it as is because we would likely have to rename it in other places too and that's not trivial as long as we want to make things clear, obvious and following established patterns.

I'm happy to revisit this for all instances of "Following" and "Followed" label in a separate PR.

**Pull request merged**